src/components/responsive-flyout/styles/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$overlay-header-height: 64px;
+
+$close-btn-size: 32px;
```

Comment:
Would `$bdl-btn-height` work instead of creating this?
https://github.com/box/box-ui-elements/blob/master/src/styles/constants/_buttons.scss#L2